talent/engineering-teams-descr.md

```diff
@@ -0,0 +1,38 @@
+This document describes the engineering teams at source{d} and how they interact with each other.
+
+### Machine Learning Team
+
+- The Machine Learning team is focused on providing intelligent insights into nearly all the world's Open Source code. This includes typical Big Data analysis as well as solving sophisticated Machine Learning problems. The former is conducted on Apache Spark clusters, the latter leverages extensive GPGPU acceleration on custom hardware using deep learning techniques. Often ML problems we face are unsupervised and require novel research in the domain of ML on source code (MLoSC).
```

Comment:
I saw @eiso and @vmarkovtsev are writing `MLonCode` instead of ~~`MLoSC`~~
They could confirm it

Reply:
In the last updates in the job descriptions around 3 weeks ago, they were written as MLoSC.

Reply:
:)

Reply:
Thanks @dpordomingo for asking on general-slack!